setup.py

```diff
@@ -18,13 +18,71 @@
 import setuptools
 from setuptools import setup
 from pathlib import Path
+from packaging import tags
```

Comment:
had to install this manually

Reply:
replaced it with a built-in alternative, thanks for pointing that out!